identity/app/controllers/SaveContentController.scala

```diff
@@ -88,8 +87,9 @@ class SaveContentController @Inject() ( api: IdApiClient,
           case (formWithErrors, Error(message, decription, _, context)) =>
             formWithErrors.withError(context.getOrElse(""), message)
         }
-        val pageData = pageDataBuilder(List.empty, emptyArticles, idRequest, pageNum)
-        Future.successful(NoCache(Ok(views.html.profile.savedForLaterPage(page, formWithErrors, pageData))))
+        pageDataBuilder(emptyArticles, idRequest, pageNum).map { pageData =>
```

Comment:
While we are in here, can we change `emptyArticles` to use `()` at the end to show it's a method call? That or change the name.